content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -22,77 +22,151 @@ The next sections assume the root folder to be `T:/o3de` but it can be in any fo
 
 The team needs to create a shared directory that all the team members can access over the network. ACS should work between Windows and Linux, but the `cacheServerAddress` string only accepts a single string such as `T:/o3de/Transfer/Cache` that all the clients are meant to read from. It's possible to have an Asset Processor in ACS mode write to one configured `cacheServerAddress` string via Windows and have another ACS retrieve from a Samba link on Linux. Your team's IT manager can provide a solution for this scenario.
 
-## Run Asset Processor in ACS mode as a server
+## Using the Shared Cache Tab
+
+The Asset Processor has a tab that manages the settings for the Shared Cache (aka Asset Cache Server).
+
+The panel is broken up into three steps:
+ 1. Choose a mode via "Set shared cache mode"
+ 2. Select the transfer directory via "Select a remote folder"
+ 3. Select the asset patterns that will be cached via "Manage shared cache patterns"
+
+![Create a log tab in Asset Processor](/images/user-guide/assets/asset-processor/acs_snapshot.png)
+
+### Choose a Mode
+
+The Shared Cache mode can be set to:
+ 1. Inactive - The Shared Cache is not active. All the assets will be processed locally.
+ 2. Server - The Asset Processor will archive product assets on a remote folder.
+ 3. Client - The Asset Processor will attempt to retrieve archive product assets from a remote folder.
+ 
+The system defaults to Inactive.
+
+### Select a Transfer Directory
+
+This field should have the full path to the remote folder that was set up in the "Create the transfer directory" step. When the "Save Changes" button is clicked, the system will detect the validity of the transfer directory.
+
+### Manage Shared Cache Patterns
+
+The table shows the active asset patterns that will be cached when the asset is processed on the server. The asset pattern row contains:
+ 1. a check box to enable or disable the pattern
+ 2. a text box to store the pattern's name
+ 3. a combo box how to use the pattern to match source assets (Wildcard or RegEx)
+ 4. a text box to enter the pattern to match source assets
+ 5. a button to remove the pattern
+ 
+The enable flag allows a user to toggle the row to disabled to force the local client Asset Processor to process the matching assets locally. Most of the time the client users will have all patterns enabled.
```

Comment:
Minor suggestion. Saying the local server processes the assets locally seems redundant.

```suggestion
The enable flag allows a user to toggle the row to disabled to force the local client Asset Processor to process the matching assets. Most of the time the client users will have all patterns enabled.
```